2025-06-20 14:43:10,789 INFO    MainThread:1684 [wandb_setup.py:_flush():70] Current SDK version is 0.19.11
2025-06-20 14:43:10,789 INFO    MainThread:1684 [wandb_setup.py:_flush():70] Configure stats pid to 1684
2025-06-20 14:43:10,789 INFO    MainThread:1684 [wandb_setup.py:_flush():70] Loading settings from /Users/lawrencebinding/.config/wandb/settings
2025-06-20 14:43:10,789 INFO    MainThread:1684 [wandb_setup.py:_flush():70] Loading settings from /Users/lawrencebinding/Desktop/projects/github/LDM-FDG/LDM-FDG/wandb/settings
2025-06-20 14:43:10,789 INFO    MainThread:1684 [wandb_setup.py:_flush():70] Loading settings from environment variables
2025-06-20 14:43:10,789 INFO    MainThread:1684 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /Users/lawrencebinding/Desktop/projects/github/LDM-FDG/LDM-FDG/wandb/run-20250620_144310-tk3a0cx0/logs/debug.log
2025-06-20 14:43:10,789 INFO    MainThread:1684 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /Users/lawrencebinding/Desktop/projects/github/LDM-FDG/LDM-FDG/wandb/run-20250620_144310-tk3a0cx0/logs/debug-internal.log
2025-06-20 14:43:10,790 INFO    MainThread:1684 [wandb_init.py:monkeypatch_ipython():643] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x103e99070>
2025-06-20 14:43:10,790 INFO    MainThread:1684 [wandb_init.py:init():852] calling init triggers
2025-06-20 14:43:10,790 INFO    MainThread:1684 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'learning_rate': 5e-05, 'architecture': 'Autoencoder_KL', 'dataset': 'FDG_2D_slices', 'epochs': 100, 'batch_size': 8, '_wandb': {}}
2025-06-20 14:43:10,790 INFO    MainThread:1684 [wandb_init.py:init():893] starting backend
2025-06-20 14:43:10,790 INFO    MainThread:1684 [wandb_init.py:init():897] sending inform_init request
2025-06-20 14:43:10,832 INFO    MainThread:1684 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-06-20 14:43:10,833 INFO    MainThread:1684 [wandb_init.py:init():907] backend started and connected
2025-06-20 14:43:10,841 INFO    MainThread:1684 [wandb_run.py:_label_probe_notebook():1376] probe notebook
2025-06-20 14:43:10,842 INFO    MainThread:1684 [wandb_run.py:_label_probe_notebook():1386] Unable to probe notebook: Expecting value: line 1 column 1 (char 0)
2025-06-20 14:43:10,842 INFO    MainThread:1684 [wandb_init.py:init():1005] updated telemetry
2025-06-20 14:43:10,892 INFO    MainThread:1684 [wandb_init.py:init():1029] communicating run to backend with 90.0 second timeout
2025-06-20 14:43:11,207 INFO    MainThread:1684 [wandb_init.py:init():1104] starting run threads in backend
2025-06-20 14:43:11,699 INFO    MainThread:1684 [wandb_run.py:_console_start():2573] atexit reg
2025-06-20 14:43:11,699 INFO    MainThread:1684 [wandb_run.py:_redirect():2421] redirect: wrap_raw
2025-06-20 14:43:11,699 INFO    MainThread:1684 [wandb_run.py:_redirect():2490] Wrapping output streams.
2025-06-20 14:43:11,699 INFO    MainThread:1684 [wandb_run.py:_redirect():2513] Redirects installed.
2025-06-20 14:43:11,702 INFO    MainThread:1684 [wandb_init.py:init():1150] run started, returning control to user process
2025-06-20 14:43:12,112 INFO    MainThread:1684 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-06-20 14:43:31,474 INFO    MainThread:1684 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-06-20 14:43:31,474 INFO    MainThread:1684 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-06-20 14:43:31,483 INFO    MainThread:1684 [wandb_run.py:_finish():2321] finishing run lbinding-ucl/LDM_FDG_vae/tk3a0cx0
2025-06-20 14:43:31,520 INFO    MainThread:1684 [jupyter.py:save_history():477] not saving jupyter history
2025-06-20 14:43:31,520 INFO    MainThread:1684 [jupyter.py:save_ipynb():401] not saving jupyter notebook
2025-06-20 14:43:31,520 INFO    MainThread:1684 [wandb_init.py:_jupyter_teardown():628] cleaning up jupyter logic
2025-06-20 14:43:31,520 INFO    MainThread:1684 [wandb_run.py:_atexit_cleanup():2538] got exitcode: 0
2025-06-20 14:43:31,520 INFO    MainThread:1684 [wandb_run.py:_restore():2520] restore
2025-06-20 14:43:31,520 INFO    MainThread:1684 [wandb_run.py:_restore():2526] restore done
2025-06-20 14:43:32,271 INFO    MainThread:1684 [wandb_run.py:_footer_history_summary_info():4188] rendering history
2025-06-20 14:43:32,271 INFO    MainThread:1684 [wandb_run.py:_footer_history_summary_info():4220] rendering summary
2025-06-20 14:43:32,271 INFO    MainThread:1684 [wandb_run.py:_footer_sync_info():4149] logging synced files
